functions/transferChatStart.ts

```diff
@@ -133,11 +133,14 @@ async function validateChannelIfWorker(
   const workerAttr = JSON.parse(worker.attributes);
 
   const unavailableVoice = channelType === 'voice' && !workerChannel.availableCapacityPercentage;
+  // if maxMessageCapacity is not set, just use configuredCapacity without adjustChatCapacity
   const unavailableChat =
-    channelType !== 'voice' &&
+    workerAttr.maxMessageCapacity 
+      ? channelType !== 'voice' &&
     !workerChannel.availableCapacityPercentage &&
     workerAttr.maxMessageCapacity &&
```

Comment:
This line is now redundant (since `workerAttr.maxMessageCapacity` must be truthy if you got here)

Reply:
Thanks for the catch!